config/initializers/retriable.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Retriable.configure do |c|
+  c.tries = 10
```

Comment:
Awesome thanks bro, could we make this configurable with an environment variable as well?